app/presenters/lcms/engine/material_presenter.rb

```diff
@@ -6,7 +6,7 @@ class MaterialPresenter < ContentPresenter
       attr_accessor :document
       attr_reader :parsed_document
 
-      delegate :css_styles, :short_url, :subject, to: :document
```

Comment:
Could you add more details about why we don't need that anymore? 😬

Reply:
we have `css_styles` at material model now.